WireAuthentication/Sources/WireAuthenticationUI/Views/DetermineAuthMethod/DetermineAuthMethodViewModel.swift

```diff
@@ -55,11 +57,15 @@ package final class DetermineAuthMethodViewModel: ObservableObject {
     package init(
         router: any Router,
         factory: any Factory,
+        backendConfig: BackendConfig?,
```

Comment:
suggestion: make this non-optional (we use the default config in case it's not on-prem), then add a bool flag to determine if it's on-prem or not so we know whether to show the header view.